assets/scss/_base.scss

```diff
@@ -0,0 +1,5 @@
+* {
```

Comment:
actually, we could go with `*, *::after, *::before` - it will apply these styles to pseudo-elements as well ;)